CHANGELOG.md

```diff
@@ -1,182 +1,275 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.13.0] - 2023-10-26
+## [0.14.0] - 2024-08-19
+
+### Changed
+
+- Updated cadCAD dependency to v0.5.3 to ensure compatibility with Python 3.10
+- Expanded Python version compatibility to include testing for Python 3.11 and 3.12
+
+### Fixed    
+
+- Resolved comp   atibility issues with Python 3.10
+
+### Maintenance
```

Comment:
This should be included under "Changed" or "Added". "Maintenance" isn't a heading in the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) standard I don't think.